posthog/errors.py

```diff
@@ -71,6 +75,19 @@ def look_up_error_code_meta(error: ServerException) -> ErrorCodeMeta:
     return CLICKHOUSE_ERROR_CODE_LOOKUP[code]
 
 
+# Specific error classes we need
+# These exist here and are not dynamically created because they are used in the codebase.
+class CHQueryErrorTooManySimultaneousQueries(InternalCHQueryError):
+    pass
+
+
+CLICKHOUSE_SPECIFIC_ERROR_LOOKUP = {
```

Comment:
Would also be good to move `EstimatedQueryExecutionTimeTooLong` and into this `QuerySizeExceeded`, in a separate PR